node/network/bridge/Cargo.toml

```diff
@@ -15,6 +15,7 @@ sc-network = { git = "https://github.com/paritytech/substrate", branch = "master
 sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "master" }
 polkadot-subsystem = { package = "polkadot-node-subsystem", path = "../../subsystem" }
 polkadot-node-network-protocol = { path = "../protocol" }
+libp2p = { version = "0.28.1", default-features = false }
```

Comment:
We're normally re-exporting everything needed through `sc-network`

```suggestion
```

Reply:
I checked it, but have overseen the `multiaddr` re-export :facepalm: